### PR TITLE
develop class based React components

### PR DESCRIPTION
**Pull request by selma-belhadj**
## What does this PR do?
This PR is raised to create the app interface and structure

## Description of the task to be completed
- create calculator class components
- create calculator UI

## Preview
[Live Demo Link](https://selmab-math-magicians.netlify.app/)

## How should it be tested
- clone the repository
`git clone https://github.com/selma-belhadj/math-magicians.git`
- navigate to the folder
`cd math-magicians`
- checkout the branch
`git checkout setup`
- Install packages
`npm install`
- Run the app
`npm start`

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```
Comment (Lameck1) [inline src/App.js]: - [x] The project requirements expects that we implement this as a class component 👇🏿:
  > - Use class based components.

  Please refactor this component to be a class-based component, not funtional.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <Button btnName="–" className="operator" />
+    <Button btnName="1" className="number" />
+    <Button btnName="2" className="number" />
+    <Button btnName="3" className="number" />
+    <Button btnName="+" className="operator" />
+    <Button btnName="0" className="number span-two" />
+    <Button btnName="." className="operation" />
+    <Button btnName="=" className="operator" />
+  </div>
+);
```
Comment (Lameck1) [inline src/components/Calculator.js]: - [x] Same issue as above. Please make this component class-based too. All components in this milestone should be class-based.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @selma-belhadj, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @selma-belhadj 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly include linters in this project and ensure that they pass before submitting for a review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by selma-belhadj** [comment]
Hello @Lameck1, thank you for you review. I have just solved the linters issue and now they are running successfully.

**Review by DeliceLydia — APPROVED** [review]
# Congratulations! 🎉🎉

## Hi @selma-belhadj, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**